steam/ext/csgo/backpack.py

```diff
@@ -110,12 +110,21 @@ async def contents(self) -> list[ProtoItem]:
                     break
         return contained_items
 
-    async def inspect(
-        self,
-        owner: SteamID,
-        d: str,
-    ):
-        ...
+    @property
+    def inspect_url(self) -> str | None:
+        """
+        Get inspect url of item if it `inspectable`
+        :return: Inspect url or None
+        """
+        try:
+            for action in self.actions or []:
```

Comment:
this or is unnecessary
```suggestion
            for action in self.actions:
```